framework/src/org/checkerframework/framework/util/typeinference/constraint/TIsU.java

```diff
@@ -12,6 +12,11 @@ public TIsU(AnnotatedTypeVariable typeVariable, AnnotatedTypeMirror relatedType)
         super(typeVariable, relatedType, 173);
```

Comment:
Here and for the other new constructors: how about replacing the old constructors with `this(..., false)` to avoid duplicating the magic constants, removing the chance of introducing an inconsistency.